src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/RepresentativeService.java

```diff
@@ -77,6 +77,9 @@ public Representative getRespondent2Representative(CaseData caseData) {
     }
 
```

Comment:
Fixed case document generation issue seen during 'Respond to Claim' for LiPvsLiP